### PR TITLE
feat(billboard): create billboard component for home page and related envs

### PR DESCRIPTION
**Pull request by zeckli**
### PR includes:
- Billboard UI component for home page
- Environment variables, contract ABI
- Billboard trackers

### CODE REVIEW

File: src/components/Billboard/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  max-width: 343px;
+
+  & a {
+    @mixin flex-center-all;
+
+    line-height: 0;
+
+    & img {
+      width: 100%;
+      max-height: 168px;
```
Comment (pitb2022) [inline src/components/Billboard/styles.module.css:12]: max-height: 16.5rem;  /* 168px */

Comment (pitb2022) [inline src/components/Billboard/styles.module.css:12]: max-height: 16.5rem; /* 168px */

File: src/components/Billboard/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.billboard {
+  display: block;
+  max-width: 343px;
```
Comment (pitb2022) [inline src/components/Billboard/styles.module.css:3]: max-height: 21.4375rem; /* 343px */

File: src/components/Billboard/styles.module.css (hunk 1 above)
Comment (pitb2022) [inline src/components/Billboard/styles.module.css:12]: max-height: 16.5rem; /* 168px */

**Review by pitb2022 — APPROVED** [review]

File: src/components/Billboard/styles.module.css (hunk 1 above)
Comment (zeckli) [reply src/components/Billboard/styles.module.css]: fixed

**Review by pitb2022 — APPROVED** [review]

**Pull request merged**